deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+  
+path="/CAT"
+
+
+if [ -d $path/project-thundercat ]; then
+    cd $path/project-thundercat
+    git pull
+    docker-compose -f docker-compose-dev.yml down
+    docker-compose -f docker-compose-dev.yml rebuild

```

Comment:
isn't `rebuild` supposed to be `build`?

Reply:
Yes, I assumed I saved it without verifying. Fixing it right now.